SU-SOE/stanford_soe_helper
# 7.x 2.x soe 3352

# READY FOR REVIEW

# Summary
- Fixed margin-left for articles tagged with multiple issues.

# Needed By (Date)
- Sprint end

# Criticality
- Needed for the release


# Steps to Test
- `git checkout 7.x-2.x-SOE-3352`
- `drush cc all`
- go to `magazine/future-everything` and make sure there is `margin-left` on multiple issue links as shown.

**Before:** 
![screen shot 2018-08-24 at 9 27 59 am](https://user-images.githubusercontent.com/571421/44590169-184aae00-a780-11e8-81bb-9eeef0066e74.png)

**After**
![screen shot 2018-08-24 at 9 28 04 am](https://user-images.githubusercontent.com/571421/44590187-21d41600-a780-11e8-808a-542e7db749db.png)


# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-3338

## Related PRs

## More Information

## Folks to notify
@cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
